### PR TITLE
Website Content Updates

### PR DESCRIPTION
**Pull request by hu3man**
Minor fixes to the docs and site

### CODE REVIEW

File: taqueria-vscode-extension/README.md [hunk 1]
```diff
@@ -29,9 +29,9 @@ A basic workflow Taqueria is:
 
 1. Initialize the project by running the `taq init` command
 2. Install necessary plugins (LIGO, SmarPy, Flextesa, Taquito, ect)
-3. Complile contracts
-4. Launch and deploy to a local sandbox
-5. Originate to a testnet
+3. Complile the source code of your smart contracts
```
Comment (jchenche) [inline taqueria-vscode-extension/README.md:32]: Typo in "Complile"

File: taqueria-vscode-extension/README.md [hunk 2]
```diff
@@ -29,9 +29,9 @@ A basic workflow Taqueria is:
...
+4. Launch a local sandbox an originate contracts to it
```
Comment (jchenche) [inline taqueria-vscode-extension/README.md:33]: Typo in "an"

**Review by jchenche — APPROVED** [review]
Aside from 2 typos, rest looks good!

**Comment by hu3man** [comment]
> 

Thanks @jchenche!

**Review by russellmorton — APPROVED** [review]

**Pull request merged**